bootstrap/kubeadm/controllers/kubeadmconfig_controller_reconciler_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
strange, it reports error that `Package sigs.k8s.io/controller-runtime/pkg/runtime/log is deprecated: use pkg/log `, but doesn't show error on `https://github.com/kubernetes-sigs/cluster-api/blob/94f8cfa757bc29e3937e75c73010f4e38db675f1/bootstrap/kubeadm/internal/locking/control_plane_init_mutex_test.go#L37`